#ifndef __HGuideAPI_save_configuration_t_h__
#define __HGuideAPI_save_configuration_t_h__
#pragma once

//Save configuration types
enum save_configuration_t
{
	configuration_Not_Save     = 0,
	configuration_Save         = 1,
	configuration_Bad_Checksum = 4
};

#endif // __HGuideAPI_save_configuration_t_h__
